src/Elastic.Apm/Api/Http.cs

```diff
@@ -9,12 +10,108 @@ namespace Elastic.Apm.Api
 	/// </summary>
 	public class Http
 	{
+		private string _url;
+
 		[JsonConverter(typeof(TrimmedStringJsonConverter))]
 		public string Method { get; set; }
 
 		[JsonProperty("status_code")]
 		public int StatusCode { get; set; }
 
-		public string Url { get; set; }
+		public string Url
```

Comment:
Maybe add docs to this as well, noting that the setter will parse and sanitise the value - and that SetUrl should be preferred if the overhead is unacceptable?

Reply:
👍  good idea, I added docs to that property.